README.rst

```diff
@@ -13,41 +13,116 @@ cookiecutter-chrisapp |Logo|
 .. |CI| image:: https://github.com/FNNDSC/cookiecutter-chrisapp/workflows/CI/badge.svg
   :alt: Github Actions
 
-A cookiecutter template for ChRIS plugin apps.
+This repo provides a cookiecutter template for ChRIS plugin apps. Using this cookiecutter will allow you to *easily* create ChRIS plugins using a ``python`` template, with baked in support for easy containerization. Recommended additional steps guide you through setting up a github repo for your plugin, triggering automatic containerization of your plugin, and automatically publishing to ``dockerhub`` and a ``chrisstore``.
+
+Note that though the plugin skeleton itself is a ``python`` construct, your application itself *does not have to be python*. Support for non-python applications is a more advanced topic with help provided elsewhere. Using this ``cookiecutter`` will require at the very minimum some python boilerplate to wrap around your application if it is non-python.
 
 Background
 ==========
 
 Before we begin, you should be familiar with these topics:
 
-* git
-* docker
-* python
+* ``git``
+* ``docker``
+* ``python``
 
 For new developers, we recommend the following reading (seasoned developers can skip this):
 
 * `What is a ChRIS plugin? <https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/About-Plugins#what-is-a-chris-plugin>`
 * `Introduction to Docker <https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Introduction-to-Docker>`
 
+Also, there is an *implicit* assumption in this documentation that you are on some OS that provides some flavour of a POSIX command line shell (``bash`` or ``zsh`` are recommended) on some UN*X env (typically some Linux distro or macOS). Please note that if you are on Windows, we have not tested these instructions on a default windows console (or PowerShell) and YMMV. We typically don't recommend using Windows for running/hosting a ChRIS system, but it could be used for developing plugins.
+
+What will this cookiecutter do?
+===============================
+
+In a nutshell this cookiecutter, will at the very minimum:
+
+* create a fully formed, but *empty* ChRIS plugin on your local filesystem (created in the directory you run the ``cookiecutter``);
+* provide a Dockerfile that you can use to create containers of your plugin;
+* provide some testing, dependency checking, and documentation files for you;
+
+We also provide additional <https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Quickstart> and instructions (and encouragement) for you to
```

Comment:
Our quickstart guide does not explain dev-ops. Dev-ops is a advanced topic. I think regular uses should be uploading their plugins in the browser at https://chrisstore.co/create

Reply:
ok... topic then for future edits. I'm going to go ahead and merge...